src/main/java/org/vanilladb/core/latch/NormalLatch.java

```diff
@@ -0,0 +1,26 @@
+package org.vanilladb.core.latch;
+
+import java.util.concurrent.locks.ReentrantLock;
+
+public class NormalLatch extends Latch{

```

Comment:
To replace `NormalLock` with `ReentrantLock` , we should replace `ReadWriteLatch` with `ReentrantReadWriteLock` at the same time, otherwise these two naming would be inconsistent. ?

Reply:
Great suggestion, let's change both of them.
by the way, I mean Reentrant***Latch***.